MainWindow.xaml.cs

```diff
@@ -81,5 +85,95 @@ private void btnLogin_Click(object sender, RoutedEventArgs e)
                 // Do something with the data
             }
         }
+        
+        private bool ValidateConfigXml(out string connStr)
```

Comment:
You can change return bool to return string? so that the function can be mocked.